sdk/nodejs/runtime/invoke.ts

```diff
@@ -231,6 +232,9 @@ async function invokeAsync(
     // Wait for all values to be available, and then perform the RPC.
     const done = rpcKeepAlive();
     try {
+        // Wait for any explicit dependencies to complete before proceeding.
+        await gatherExplicitDependencies(opts.dependsOn);
```

Comment:
Dumb typescript question, if only `InvokeOptions` is passed in here `opts.dependsOn` will be `undefined`? `gatherExplicitDependencies` won't do anything with that and that's how this works?